jdaviz/configs/cubeviz/plugins/slice/slice.vue

```diff
@@ -14,16 +14,16 @@
             <v-row>
               <v-switch
                 label="Show Indicator"
-                hint="Show indicator in spectral viewer even when slice tool is inactive."
+                hint="Show slice indicator even when slice tool is inactive."
                 v-model="show_indicator"
                 persistent-hint>
               </v-switch>
             </v-row>
             <v-row>
               <v-switch
-                label="Show Wavelength"
-                hint="Show slice wavelength in label to right of indicator."
-                v-model="show_wavelength"
+                label="Show Value"
+                :hint="'Show slice '+value_label.toLowerCase()+' in label to right of indicator.'"
```

Comment:
I don't understand the need for `toLowerCase` here.

Reply:
Because we want lower case here (not "Show slice Wavelength in label"), but we want title case in the actual input widget, without defining two different strings on the python-side.

Reply:
Okay, just to make sure, this does not affect string representation of the unit, right? Because "MJy" is not the same as "mjy".

Reply:
now, this is the label (Wavelength, Frequency, Time, etc), not the unit.